node/node.go

```diff
@@ -304,6 +304,8 @@ type Node struct {
 	// Manages network timeouts
 	timeoutManager timeout.Manager
 
+	subnets *chains.Subnets
```

Comment:
Is there a reason this is added to the struct? seems like it is only used in one function right now (perhaps used in a follow-up?)

Reply:
I don't think we need to have it on the struct... if we do we can fix it in the follow-up.